www/src/pages/docs/50_StoreMapping.md

```diff
@@ -358,15 +355,9 @@ val ageLens: Lens<Person, Int> = Person.age() // cannot be used in tag attribute
 val ageLensAsString: Lens<Person, String> = Person.age().asString() // now it is usable
 ```
 
-fritz2 also provides a special `lensOf()` function for creating a `Lens<P, String>` for special types that are not 
```

Comment:
Why omit the code here? Repeitition is useful here - the documentation could be read without reading the `Overview` sections! Getting the signature here is imho a fine thing.